src/components/Calculator.js

```diff
@@ -1,33 +1,66 @@
 import React from 'react';
+import calculate from '../logic/calculate';
 import './calculator.css';
 
 // eslint-disable-next-line react/prefer-stateless-function
```

Comment:
- [ ] OPTIONAL
You have **disabled ESLint checks** on parts of your code. Please enable all ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.